build/templates/_converters.py.mako

```diff
@@ -10,8 +10,9 @@ import datetime
 def _convert_timedelta(value, library_type, scaling):
     scaled_value = value.total_seconds() * scaling
 
-    if not library_type == visatype.ViReal64 and not library_type == visatype.ViReal32:  # ctype integer types don't convert to int from float so we need to
-        scaled_value = int(scaled_value + 0.5)
+    # ctype integer types don't convert to int from float so we need to
+    if library_type in [visatype.ViInt64, visatype.ViInt32, visatype.ViUInt32, visatype.ViInt16, visatype.ViUInt16, visatype.ViInt8]:
```

Comment:
I worry that once we add, say, ViUInt8 support to the generator, we won't update this list.
It'd be great if this was single sourced. Could you look at all the types and filter only those that are mapped to an integer type?

Reply:
I'm ok if this becomes a TODO and is solved as a separate PR.

Reply:
I think it will need to be a separate PR. I don't see an simple and quick solution. Doing the `str(type())` trick doesn't work since all they all return `<type '_ctypes.PyCSimpleType'>`, including float and double.